controller_manager/controller_manager/spawner.py

```diff
@@ -25,9 +24,13 @@
     load_controller, switch_controllers, unload_controller
 
 import rclpy
+from rcl_interfaces.msg import Parameter
 from rclpy.duration import Duration
 from rclpy.node import Node
+from rclpy.parameter import get_parameter_value
 from rclpy.signals import SignalHandlerOptions
+from ros2param.api import call_set_parameters
+from ros2param.api import load_parameter_file
```

Comment:
comparing [humble](https://github.com/ros2/ros2cli/blob/humble/ros2param/ros2param/verb/set.py#L25) and [rolling](https://github.com/ros2/ros2cli/blob/rolling/ros2param/ros2param/verb/set.py#L18) this should fix the issue.

```suggestion
from rcl_interfaces.msg import Parameter
from rclpy.duration import Duration
from rclpy.node import Node
from rclpy.signals import SignalHandlerOptions
from ros2param.api import call_set_parameters
from ros2param.api import get_parameter_value
from ros2param.api import load_parameter_file
```